operator/webhook/core/sync.py

```diff
@@ -426,44 +411,49 @@ def _new_actuator_service(parent):
         "metadata": {
             "labels": {
                 "integration-route": parent_metadata["name"],
-                "prometheus-metrics-enabled": "true"
+                "prometheus-metrics-enabled": "true",
             },
-            "name": f'{parent_metadata["name"]}-actuator'
+            "name": f'{parent_metadata["name"]}-actuator',
         },
         "spec": {
             "ports": [
                 {
                     "name": scheme,
                     "port": management_port,
                     "protocol": "TCP",
-                    "targetPort": management_port
+                    "targetPort": management_port,
                 }
             ],
             "selector": {
                 "app.kubernetes.io/instance": f'integrationroute-{parent_metadata["name"]}'
-            }
-        }
+            },
+        },
     }
 
     return service
 
+
 def _has_tls(parent) -> bool:
     return "tls" in parent["spec"] and "keystore" in parent["spec"]["tls"]
 
+
 def _get_scheme(has_tls) -> str:
     return "https" if has_tls else "http"
 
+
 def _get_management_port(has_tls) -> int:
     return HTTPS_PORT if has_tls else HTTP_PORT
 
+
 def _gen_children(parent) -> List[Mapping]:
     return [_new_deployment(parent), _new_actuator_service(parent)]
 
 
-def sync(parent) -> Mapping:
+def sync(body) -> Mapping:
+    # Request API at https://metacontroller.github.io/metacontroller/api/compositecontroller.html#sync-hook-request
+    _LOGGER.debug("\n\nRequest:\n%s", body)
```

Comment:
Looks like we already log the request as soon as it's received in `app.py`. We might want to avoid logging again here so we don't spam the logs with redundant info.

Reply:
Fixed

Reply:
Should we remove the other logging statement as well, that logs the reply?

Reply:
Fixed